.github/workflows/ci-core.yml

```diff
@@ -53,9 +54,29 @@ jobs:
         id: ignore-filter
         run: echo "changes=true" >> $GITHUB_OUTPUT
 
+      - name: Filter other changes
+        uses: ./.github/actions/filter-changes
+        id: filter
+        with:
+          # Filter files that match **every** pattern in this list
+          every-filter: |
+            - '**'                     # Initially include all files
+            - '!**/.github/**'         # Exclude all changes under the .github directory
+            - '!integration-tests/**'  # Exclude all changes under the integration-tests directory
+          # Filter files that match **any** of these patterns
+          any-filter: |
+            - '.github/workflows/ci-core.yml'
+            - '.github/actions/golangci-lint/**'
+            - '.github/actions/filter-changes/**'
+            - '.github/actions/setup-go/**'
+            - '.github/actions/setup-nodejs/**'
+            - '.github/actions/setup-postgres/**'
+            - '.github/actions/setup-solana/**'
+            - '.github/actions/setup-wasmd/**'
+
   golangci:
     # We don't directly merge dependabot PRs, so let's not waste the resources
-    if: ${{ (github.event_name == 'pull_request' ||  github.event_name == 'schedule') && github.actor != 'dependabot[bot]' }}
+    if: ${{ (github.event_name == 'pull_request' ||  github.event_name == 'schedule') && github.actor != 'dependabot[bot]' && needs.filter.outputs.not_ignored_changes == 'true' }}
```

Comment:
Scheduled runs are against the state of the repo, and whether or not there are recent changes is not relevant. We expect to flag all outstanding linter debt.

Reply:
@jmank88 fixed in 74493af42cd78509918887f5fced054ad65a5f52